domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/CharSequenceAssertions.kt

```diff
@@ -24,9 +24,13 @@ interface CharSequenceAssertions {
     fun <T : CharSequence> containsNotBuilder(subjectProvider: SubjectProvider<T>): CharSequenceContains.Builder<T, NotSearchBehaviour>
 
     fun startsWith(subjectProvider: SubjectProvider<CharSequence>, expected: CharSequence): Assertion
+    fun startsWith(subjectProvider: SubjectProvider<CharSequence>, expected: Char): Assertion
```

Comment:
Yapp, I still think delegation is good enough (this applies to the other methods you added here as well). Unless you see a use case where we want to treat `Char` different than `CharSequence`, in this case we can discuss it. Otherwise I think it only clutters the API without real value.

Reply:
Make sense. Will update